### ISSUE TITLE
docs: usage page (easy configuration)

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/mlem.ai#190]
MLEM has enough complication in it's setup that we probably need a wizard-like page for users to get up and running quickly on different platforms and set ups.

Examples:

- https://cml.dev/doc/usage
- https://spacy.io/usage

**Comment by aguschin** [comment]
Good idea, @jorgeorpinel. We discussed this with @shcheklein and @mike0sv. The installation is the same everywhere now (`pip install mlem`). The different is in usage scenarios. So it could be:

```
I have [ CV | Table data | NLP ] task
...<the selected part is shown>...

I want to [ serve | build | deploy ] my model
...<the selected part is shown>...
```

To enable the first selector, we need to support CV and NLP. CV is going to be implemented in Q4, and NLP will probably be implemented in 2023Q1. Rn we can enable the 2nd selector only.

I plan to simplify GS in #188 now, but still keep the usage scenarios (build/serve/deploy) on separate pages. Once we support CV, it should be a good moment to get back to this and see how your suggestion may look like.

**Comment by aguschin** [comment]
@jorgeorpinel, reading your description update in here https://github.com/iterative/mlem.ai/issues/129:

> Truncate code block (link to full file in example repo).
> Show relevant parts of the .mlem file (link to full ").

At the same time in this issue you write:
> to get up and running quickly

To my mind, the latest means that one should be able to copy-paste code and get a working example. To later modify it for one's needs. Then these two motivations are in contradiction with each other. Could you please clarify?

**Comment by jorgeorpinel** [comment]
> > get up and running quickly
> means that one should be able to copy-paste code and get a working example

Yes but I didn't suggest the quick usage page would be integrated into the Get Started. I imagine it would be it's own page or some sort of separate widget, or part of https://mlem.ai/doc/install.

**Comment by aguschin** [comment]
For the history: there was an attempt for that in https://github.com/iterative/mlem.ai/pull/246, but we decided it's not the right time to do it now. We can get back to this later.

**Comment by jorgeorpinel** [comment]
> tries to achieve two things: show use cases and provide code templates

How else can templates be organized (if not by use case) ?

> put it in different pages

Adding templates everywhere is another possible strategy indeed. Currently docs are very descriptive and not copy-paste-able enough, in general.

> have a diagram like in https://lets-unify.ai

Any good diagrams you come up with even if they're hand drawn please share them and we'll see where they can be useful. An image can truly = 1000 words!

**Comment by aguschin** [comment]
I had some ideas in https://miro.com/app/board/uXjVP5qFYLg=/, maybe they will be useful at some point.
